raidnearby.py

```diff
@@ -758,21 +787,10 @@ def processRaidImage(self, raidfilename):
 
     def send_webhook(self, fort_id, lvl, poke_id, end, type_):
         battle_start = end
-        if fort_id in wh_send_dict:
-            if wh_send_dict[fort_id]["poke_id"] == poke_id:
-                LOG.info("Webhook: Raid was already send earlier")
-                return
-            else:
-                external_id = wh_send_dict[fort_id]["ext_id"]
-                lat = wh_send_dict[fort_id]["lat"]
-                lon = wh_send_dict[fort_id]["lon"]
-                name = wh_send_dict[fort_id]["name"]
-                sponsor = wh_send_dict[fort_id]["sponsor"]
-                move_1 = wh_send_dict[fort_id]["move_1"]
-                move_2 = wh_send_dict[fort_id]["move_2"]
-                cp = wh_send_dict[fort_id]["cp"]
-                form = wh_send_dict[fort_id]["form"]
-                wh_send_dict[fort_id]["poke_id"] = poke_id
+        #Need to resend the webook if changing from egg to raid, or if raid is from a different time
+        if fort_id in wh_send_dict and wh_send_dict[fort_id]["end"] == end and wh_send_dict[fort_id]["type"] == type_:		
```

Comment:
`end` is not in the `wh_send_dict` this would lead to a KeyError

you first have to add it in line 807-817

Reply:
Thanks. I missed that when I was copying my changes to the github clone